simpleprophet/simpleprophet/models.py

```diff
@@ -202,7 +196,11 @@ def data_filter(data, product):
         "Firefox iOS Global MAU": [s2d('2017-11-08'), s2d('2017-12-31')],
         "Firefox iOS Tier1 MAU": [s2d('2017-11-08'), s2d('2017-12-31')],
         "Mobile Global MAU": [s2d('2017-11-10'), s2d('2018-03-11')],
+        "Mobile Global MAU": [s2d('2019-12-04'), s2d('2020-01-27')],
+        "Mobile Global MAU": [s2d('2020-08-01'), s2d('2020-10-08')],
```

Comment:
Should we add a comment alongside each outlier patch (going forward), explaining what's being removed?

E.g. 
```suggestion
        "Mobile Global MAU": [s2d('2020-08-01'), s2d('2020-10-08')],  # Fennec -> Fenix transition
```

Reply:
sg, just added to the comment

Reply:
> I agree with Felix that we should have some commentary in the code to mention that these parameters changed in Nov 2020 and give a brief explanation. Perhaps also provide a link to a more complete explanation.

Just added some details for the reason of parameter change in the code. Can you take a look?